Jenkinsfile

```diff
@@ -632,6 +638,72 @@ pipeline {
       }
     }
 
+    stage('Deploy Preview') {
+      when {
+        not {
+          branch 'PR-*'
+        }
+      }
+      steps {
+        setGitHubBuildStatus('nuxeo/preview', 'Deploy nuxeo Preview', 'PENDING')
+        container('maven') {
+          dir('ci/helm/preview') {
+            echo """
+            ----------------------------------------
+            Deploy Preview environment
+            ----------------------------------------"""
+            // first substitute environment variables in chart values
+            sh """
+              mv values.yaml values.yaml.tosubst
+              envsubst < values.yaml.tosubst > values.yaml
+            """
+            // second create target namespace (if doesn't exist) and copy secrets to target namespace
+            script {
+              boolean nsExist = sh(returnStatus: true, script: "kubectl get namespace ${PREVIEW_NAMESPACE}") == 0
+              String noCommentOpt = '';
+              if (nsExist) {
+                noCommentOpt = '--no-comment'
+              } else {
+                sh "kubectl create namespace ${PREVIEW_NAMESPACE}"
+              }
+              sh "kubectl create secret generic preview-docker-cfg --namespace=${PREVIEW_NAMESPACE} --from-file=.dockerconfigjson=/home/jenkins/.docker/config.json --type=kubernetes.io/dockerconfigjson --dry-run -o yaml | kubectl apply -f -"
+              // third build and deploy the chart
+              // reference branch have a different preview command. The -n name provided must be lowercase.
+              previewCommand = "jx step helm install --namespace ${PREVIEW_NAMESPACE} -n ${APP_NAME}-${PREVIEW_NAMESPACE.toLowerCase()} ."
+              if (isPullRequest()) {
+                // waiting for https://github.com/jenkins-x/jx/issues/5797 to be fixed in order to remove --source-url
+                previewCommand = "jx preview --namespace ${PREVIEW_NAMESPACE} --source-url=https://github.com/nuxeo/nuxeo ${noCommentOpt}"
+              }
+              sh """
+                jx step helm build
```

Comment:
Same remark as https://github.com/nuxeo/nuxeo/pull/3856/commits/a56c1daccaf38394bed76fb5471085cf33646b1d#r401422462: would `helm build` work?

Reply:
Maybe this would make the direct helm install work (cf https://github.com/nuxeo/nuxeo/pull/3856#discussion_r401431268), I'm gonna test it

Reply:
Don't think so, but yes you can give it a try.

Reply:
Indeed `helm build` doesn't exist jx stem helm build wraps some stuff https://github.com/jenkins-x/jx/blob/8fdc3a1182bc3ed6ae09728721b53ea4fa1d6234/pkg/cmd/step/helm/step_helm_build.go that would need further investigation to see how to translate it in helm commands.

I'm sticking with jx step helm install as you suggest https://github.com/nuxeo/nuxeo/pull/3856#discussion_r401443689

Reply:
:+1: 
Yes, the equivalent is `helm package`.